src/test/java/HomeWork_20.java

```diff
@@ -0,0 +1,66 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork_20 extends BaseTest {
+    @Test
+    public void successfulLoginTest() {
+        login("chernyakma75@gmail.com", "te$t$tudent");
+        WebElement avatar = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("a .avatar")));
+        Assert.assertTrue(avatar.isDisplayed());
+    }
+    @Test
+    public void wrongPasswordLoginTest() {
+        login("chernyakma75@gmail.com", "te$t$tuden");
+        WebElement submitLoginButton = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("button[type='submit']")));
+        Assert.assertTrue(submitLoginButton.isDisplayed());
+    }
+
+    @Test
+    public void emptyPasswordLoginTest() {
+        login("chernyakma75@gmail.com", "");
+        WebElement submitLoginButton = wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("button[type='submit']")));
+        Assert.assertTrue(submitLoginButton.isDisplayed());
+    }
+    @Test
+    public void RegistrationNovigation() {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example public void **registrationNovigation()**